requirements/poseval.txt

```diff
@@ -1 +1,2 @@
 poseval@git+https://github.com/svenkreiss/poseval.git
+shapely==1.8.4
```

Comment:
Is `shapely` required by `poseval`?

Reply:
yes

Reply:
It seems already in the requirements of `poseval` (https://github.com/svenkreiss/poseval/blob/master/setup.py#L15). Won't it be automatically installed?

Reply:
There is a new release of `shapely` today (v1.8.5), which will be automatically installed. This version of `shapely` seems incompatible with the unittest environment, and will make CI fail.